tests/by-util/test_base64.rs

```diff
@@ -40,6 +40,49 @@ fn test_encode_repeat_flags_later_wrap_15() {
         .stdout_only("aGVsbG8sIHdvcmx\nkIQ==\n"); // spell-checker:disable-line
 }
 
+// Different behavior between GNU 9.4 & 9.5
+// See GNU 8c735f65857ea17cdc0218b3fa3223769c3d8b9c
+#[test]
+fn test_code_short() {
```

Comment:
Is `code` in the function name on purpose or should it be `decode`?

Reply:
actually, i have been able to get the error back